applications/luci-app-mwan3/htdocs/luci-static/resources/view/mwan3/mwan3.js

```diff
@@ -0,0 +1,95 @@
+function secondsToString(time) {
```

Comment:
You should be able to replace every `secondsToString(val)`  call with `'%t'.format(val)` and subsequently drop this function